### PR TITLE
Fix/cleanup templates, no functional changes

### PR DESCRIPTION
**Pull request by sikmir**
### **config-cityron-pu-3**: fix title, convert to jinja

<details>
<summary>Validation</summary>
<code>
$ git worktree add ../wb-mqtt-serial-master master
$ j2 -o templates/config-cityron-pu-3.json templates/config-cityron-pu-3.json.jinja
$ diff -Naur ../wb-mqtt-serial-master/templates/config-cityron-pu-3.json templates/config-cityron-pu-3.json
--- ../wb-mqtt-serial-master/templates/config-cityron-pu-3.json	2023-11-15 03:06:02.000000000 +0400
+++ templates/config-cityron-pu-3.json	2023-11-15 04:25:38.000000000 +0400
@@ -1,9 +1,9 @@
 {
-    "title": "Cityron ПУ-3",
+    "title": "Cityron PU-3",
     "device_type": "cityron_pu-3",
     "group": "g-control-panel",
     "device": {
-        "name": "Cityron ПУ-3",
+        "name": "Cityron PU-3",
         "id": "cityron-pu-3",
         "response_timeout_ms": 100,
         "guard_interval_us": 5000,
@@ -370,9 +370,10 @@
                 "group": "data_storage",
                 "enabled": false
             }
-        ],
+            ],
         "translations": {
             "ru": {
+                "Cityron PU-3": "Cityron ПУ-3",
                 "Parameters": "Параметры",
                 "General": "Общее",
                 "Data Storage": "Хранилище данных",
@@ -414,7 +415,7 @@
                 "Register 407": "Регистр 407",
                 "Register 408": "Регистр 408",
                 "Register 409": "Регистр 409"
-            }
+                }
         }
     }
-}
+}
\ No newline at end of file
</code>
</details>

### **config-ddm845r**: convert to jinja

<details>
<summary>Validation</summary>
<code>
$ j2 -o templates/config-ddm845r.json templates/config-ddm845r.json.jinja
$ diff -Naurw ../wb-mqtt-serial-master/templates/config-ddm845r.json templates/config-ddm845r.json
--- ../wb-mqtt-serial-master/templates/config-ddm845r.json	2023-11-15 03:06:02.000000000 +0400
+++ templates/config-ddm845r.json	2023-11-15 04:29:09.000000000 +0400
@@ -98,7 +98,6 @@
         "address": "60",
         "scale": 0.01,
 	"round_to": 1.0
-
       }
     ]
   }
</code>
</details>

### **config-ecodim-dali-gw2**: refactor

<details>
<summary>Validation</summary>
<code>
$ j2 -o templates/config-ecodim-dali-gw2.json templates/config-ecodim-dali-gw2.json.jinja
$ j2 -o ../wb-mqtt-serial-master/templates/config-ecodim-dali-gw2.json ../wb-mqtt-serial-master/templates/config-ecodim-dali-gw2.json.jinja
$ diff -Naur ../wb-mqtt-serial-master/templates/config-ecodim-dali-gw2.json templates/config-ecodim-dali-gw2.json
--- ../wb-mqtt-serial-master/templates/config-ecodim-dali-gw2.json	2023-11-15 04:06:16.000000000 +0400
+++ templates/config-ecodim-dali-gw2.json	2023-11-15 04:33:31.000000000 +0400
@@ -223,90 +223,99 @@
             },
             {
                 "name": "Group 0 Devices Available",
-                "address": 2002,
+                "address": "2002",
                 "reg_type": "holding",
                 "type": "switch",
                 "format": "u16",
                 "scale": 0.5,
                 "readonly": true,
-                "group": "group_commands"
+                "group": "group_commands",
+                "enabled": true
             },
             {
                 "name": "Group 0 Fault",
-                "address": 2003,
+                "address": "2003",
                 "reg_type": "holding",
                 "type": "switch",
                 "format": "u16",
                 "scale": 0.5,
                 "readonly": true,
-                "group": "group_commands"
+                "group": "group_commands",
+                "enabled": true
             },
             {
                 "name": "Group 0 Status",
-                "address": 2004,
+                "address": "2004",
                 "reg_type": "holding",
                 "type": "switch",
                 "format": "u16",
                 "scale": 0.5,
                 "readonly": true,
-                "group": "group_commands"
+                "group": "group_commands",
+                "enabled": true
             },
             {
                 "name": "Group 0 Adjustable Color Temperature Devices",
-                "address": 2622,
+                "address": "2622",
                 "reg_type": "holding",
                 "type": "switch",
                 "format": "u16",
                 "scale": 0.5,
                 "readonly": true,
-                "group": "group_commands"
+                "group": "group_commands",
+                "enabled": true
             },
             {
                 "name": "Group 0 Command",
-                "write_address": 2001,
+                "write_address": "2001",
                 "reg_type": "holding",
                 "type": "value",
                 "format": "u16",
                 "group": "group_commands",
+                "enabled": true,
                 "min": 0,
                 "max": 31
             },
             {
                 "name": "Group 0 Brightness",
-                "write_address": 2000,
+                "write_address": "2000",
                 "reg_type": "holding",
                 "type": "range",
                 "format": "u16",
                 "group": "group_commands",
+                "enabled": true,
                 "min": 0,
                 "max": 254
             },
             {
                 "name": "Group 0 Color Temperature",
-                "write_address": 2620,
+                "write_address": "2620",
                 "reg_type": "holding",
                 "type": "range",
                 "format": "u16",
                 "group": "group_commands",
+                "enabled": true,
                 "min": 1000,
                 "max": 10000
             },
             {
                 "name": "Group 0 Increase Color Temperature",
-                "write_address": 2621,
+                "write_address": "2621",
                 "reg_type": "holding",
                 "type": "pushbutton",
                 "format": "u16",
                 "group": "group_commands",
+                "enabled": true,
                 "on_value": 0
             },
             {
                 "name": "Group 0 Decrease Color Temperature",
-                "write_address": 2621,
+                "write_address": "2621",
                 "reg_type": "holding",
                 "type": "pushbutton",
                 "format": "u16",
                 "group": "group_commands",
+                "enabled": true,
                 "on_value": 1
             },
             {
@@ -1766,25 +1775,27 @@
             },
             {
                 "name": "Device 0 Available",
-                "address": 3002,
+                "address": "3002",
                 "reg_type": "holding",
                 "type": "value",
                 "format": "u16",
                 "readonly": true,
-                "group": "individual_commands"
+                "group": "individual_commands",
+                "enabled": true
             },
             {
                 "name": "Device 0 Status",
-                "address": 3003,
+                "address": "3003",
                 "reg_type": "holding",
                 "type": "value",
                 "format": "u16",
                 "readonly": true,
-                "group": "individual_commands"
+                "group": "individual_commands",
+                "enabled": true
             },
             {
                 "name": "Device 0 Color Control Mode",
-                "address": 3322,
+                "address": "3322",
                 "reg_type": "holding",
                 "type": "value",
                 "format": "u16",
@@ -1794,7 +1805,7 @@
             },
             {
                 "name": "Device 0 Color Change Status",
-                "address": 3323 ,
+                "address": "3323",
                 "reg_type": "holding",
                 "type": "value",
                 "format": "u16",
@@ -1804,29 +1815,31 @@
             },
             {
                 "name": "Device 0 Command",
-                "write_address": 3001,
+                "write_address": "3001",
                 "reg_type": "holding",
                 "type": "value",
                 "format": "u16",
                 "group": "individual_commands",
+                "enabled": true,
                 "min": 0,
                 "max": 31
             },
             {
                 "name": "Device 0 Brightness",
-                "address": 3004,
-                "write_address": 3000,
+                "address": "3004",
+                "write_address": "3000",
                 "reg_type": "holding",
                 "type": "range",
                 "format": "u16",
                 "group": "individual_commands",
+                "enabled": true,
                 "min": 0,
                 "max": 254
             },
             {
                 "name": "Device 0 Color Temperature",
-                "address": 3324 ,
-                "write_address": 3320,
+                "address": "3324",
+                "write_address": "3320",
                 "reg_type": "holding",
                 "type": "range",
                 "format": "u16",
@@ -1837,7 +1850,7 @@
             },
             {
                 "name": "Device 0 Increase Color Temperature",
-                "write_address": 3321,
+                "write_address": "3321",
                 "reg_type": "holding",
                 "type": "pushbutton",
                 "format": "u16",
@@ -1847,7 +1860,7 @@
             },
             {
                 "name": "Device 0 Decrease Color Temperature",
-                "write_address": 3321,
+                "write_address": "3321",
                 "reg_type": "holding",
                 "type": "pushbutton",
                 "format": "u16",
</code>
</details>

### **config-map12e-fw2**: convert to jinja

<details>
<code>
$ j2 -o templates/config-map12e-fw2.json templates/config-map12e-fw2.json.jinja
$ diff -Naur ../wb-mqtt-serial-master/templates/config-map12e-fw2.json templates/config-map12e-fw2.json
--- ../wb-mqtt-serial-master/templates/config-map12e-fw2.json	2023-11-15 03:06:02.000000000 +0400
+++ templates/config-map12e-fw2.json	2023-11-15 05:04:47.000000000 +0400
@@ -597,7 +597,7 @@
                 "round_to": 1e-6,
                 "word_order": "little_endian",
                 "group": "channel1_l1"
-            },
+            },
             {
                 "name": "Ch 1 RP energy L1",
                 "reg_type": "input",
@@ -619,7 +619,7 @@
                 "round_to": 1e-6,
                 "word_order": "little_endian",
                 "group": "channel1_l1"
-            },
+            },
             {
                 "name": "Ch 1 Phase angle L1",
                 "reg_type": "input",
@@ -712,7 +712,7 @@
                 "round_to": 1e-6,
                 "word_order": "little_endian",
                 "group": "channel1_l2"
-            },
+            },
             {
                 "name": "Ch 1 RP energy L2",
                 "reg_type": "input",
@@ -734,7 +734,7 @@
                 "round_to": 1e-6,
                 "word_order": "little_endian",
                 "group": "channel1_l2"
-            },
+            },
             {
                 "name": "Ch 1 Phase angle L2",
                 "reg_type": "input",
@@ -827,7 +827,7 @@
                 "round_to": 1e-6,
                 "word_order": "little_endian",
                 "group": "channel1_l3"
-            },
+            },
             {
                 "name": "Ch 1 RP energy L3",
                 "reg_type": "input",
@@ -849,7 +849,7 @@
                 "round_to": 1e-6,
                 "word_order": "little_endian",
                 "group": "channel1_l3"
-            },
+            },
             {
                 "name": "Ch 1 Phase angle L3",
                 "reg_type": "input",
@@ -921,7 +921,7 @@
                 "round_to": 1e-6,
                 "word_order": "little_endian",
                 "group": "channel1_total"
-            },
+            },
             {
                 "name": "Ch 1 Total RP energy",
                 "reg_type": "input",
@@ -943,7 +943,7 @@
                 "round_to": 1e-6,
                 "word_order": "little_endian",
                 "group": "channel1_total"
-            },
+            },
             {
                 "name": "Ch 2 Irms L1",
                 "reg_type": "input",
@@ -1026,7 +1026,7 @@
                 "round_to": 1e-6,
                 "word_order": "little_endian",
                 "group": "channel2_l1"
-            },
+            },
             {
                 "name": "Ch 2 RP energy L1",
                 "reg_type": "input",
@@ -1048,7 +1048,7 @@
                 "round_to": 1e-6,
                 "word_order": "little_endian",
                 "group": "channel2_l1"
-            },
+            },
             {
                 "name": "Ch 2 Phase angle L1",
                 "reg_type": "input",
@@ -1141,7 +1141,7 @@
                 "round_to": 1e-6,
                 "word_order": "little_endian",
                 "group": "channel2_l2"
-            },
+            },
             {
                 "name": "Ch 2 RP energy L2",
                 "reg_type": "input",
@@ -1163,7 +1163,7 @@
                 "round_to": 1e-6,
                 "word_order": "little_endian",
                 "group": "channel2_l2"
-            },
+            },
             {
                 "name": "Ch 2 Phase angle L2",
                 "reg_type": "input",
@@ -1256,7 +1256,7 @@
                 "round_to": 1e-6,
                 "word_order": "little_endian",
                 "group": "channel2_l3"
-            },
+            },
             {
                 "name": "Ch 2 RP energy L3",
                 "reg_type": "input",
@@ -1278,7 +1278,7 @@
                 "round_to": 1e-6,
                 "word_order": "little_endian",
                 "group": "channel2_l3"
-            },
+            },
             {
                 "name": "Ch 2 Phase angle L3",
                 "reg_type": "input",
@@ -1350,7 +1350,7 @@
                 "round_to": 1e-6,
                 "word_order": "little_endian",
                 "group": "channel2_total"
-            },
+            },
             {
                 "name": "Ch 2 Total RP energy",
                 "reg_type": "input",
@@ -1372,7 +1372,7 @@
                 "round_to": 1e-6,
                 "word_order": "little_endian",
                 "group": "channel2_total"
-            },
+            },
             {
                 "name": "Ch 3 Irms L1",
                 "reg_type": "input",
@@ -1455,7 +1455,7 @@
                 "round_to": 1e-6,
                 "word_order": "little_endian",
                 "group": "channel3_l1"
-            },
+            },
             {
                 "name": "Ch 3 RP energy L1",
                 "reg_type": "input",
@@ -1477,7 +1477,7 @@
                 "round_to": 1e-6,
                 "word_order": "little_endian",
                 "group": "channel3_l1"
-            },
+            },
             {
                 "name": "Ch 3 Phase angle L1",
                 "reg_type": "input",
@@ -1570,7 +1570,7 @@
                 "round_to": 1e-6,
                 "word_order": "little_endian",
                 "group": "channel3_l2"
-            },
+            },
             {
                 "name": "Ch 3 RP energy L2",
                 "reg_type": "input",
@@ -1592,7 +1592,7 @@
                 "round_to": 1e-6,
                 "word_order": "little_endian",
                 "group": "channel3_l2"
-            },
+            },
             {
                 "name": "Ch 3 Phase angle L2",
                 "reg_type": "input",
@@ -1685,7 +1685,7 @@
                 "round_to": 1e-6,
                 "word_order": "little_endian",
                 "group": "channel3_l3"
-            },
+            },
             {
                 "name": "Ch 3 RP energy L3",
                 "reg_type": "input",
@@ -1707,7 +1707,7 @@
                 "round_to": 1e-6,
                 "word_order": "little_endian",
                 "group": "channel3_l3"
-            },
+            },
             {
                 "name": "Ch 3 Phase angle L3",
                 "reg_type": "input",
@@ -1779,7 +1779,7 @@
                 "round_to": 1e-6,
                 "word_order": "little_endian",
                 "group": "channel3_total"
-            },
+            },
             {
                 "name": "Ch 3 Total RP energy",
                 "reg_type": "input",
@@ -1801,7 +1801,7 @@
                 "round_to": 1e-6,
                 "word_order": "little_endian",
                 "group": "channel3_total"
-            },
+            },
             {
                 "name": "Ch 4 Irms L1",
                 "reg_type": "input",
@@ -1884,7 +1884,7 @@
                 "round_to": 1e-6,
                 "word_order": "little_endian",
                 "group": "channel4_l1"
-            },
+            },
             {
                 "name": "Ch 4 RP energy L1",
                 "reg_type": "input",
@@ -1906,7 +1906,7 @@
                 "round_to": 1e-6,
                 "word_order": "little_endian",
                 "group": "channel4_l1"
-            },
+            },
             {
                 "name": "Ch 4 Phase angle L1",
                 "reg_type": "input",
@@ -1999,7 +1999,7 @@
                 "round_to": 1e-6,
                 "word_order": "little_endian",
                 "group": "channel4_l2"
-            },
+            },
             {
                 "name": "Ch 4 RP energy L2",
                 "reg_type": "input",
@@ -2021,7 +2021,7 @@
                 "round_to": 1e-6,
                 "word_order": "little_endian",
                 "group": "channel4_l2"
-            },
+            },
             {
                 "name": "Ch 4 Phase angle L2",
                 "reg_type": "input",
@@ -2114,7 +2114,7 @@
                 "round_to": 1e-6,
                 "word_order": "little_endian",
                 "group": "channel4_l3"
-            },
+            },
             {
                 "name": "Ch 4 RP energy L3",
                 "reg_type": "input",
@@ -2136,7 +2136,7 @@
                 "round_to": 1e-6,
                 "word_order": "little_endian",
                 "group": "channel4_l3"
-            },
+            },
             {
                 "name": "Ch 4 Phase angle L3",
                 "reg_type": "input",
@@ -2208,7 +2208,7 @@
                 "round_to": 1e-6,
                 "word_order": "little_endian",
                 "group": "channel4_total"
-            },
+            },
             {
                 "name": "Ch 4 Total RP energy",
                 "reg_type": "input",
@@ -2230,7 +2230,7 @@
                 "round_to": 1e-6,
                 "word_order": "little_endian",
                 "group": "channel4_total"
-            },
+            },
             {
                 "name": "Urms L1",
                 "reg_type": "input",
@@ -2287,7 +2287,7 @@
                 "error_value": "0xFFFFFFFF",
                 "group": "l2"
             },
-           {
+            {
                 "name": "U L2-L3",
                 "reg_type": "input",
                 "address": "0x141D",
@@ -2518,4 +2518,4 @@
             }
         }
     }
-}
+}
\ No newline at end of file
</code>
</details>

### **config-uniel-uch-m111rx**: convert to jinja

<details>
<summary>Validation</summary>
<code>
$ j2 -o templates/config-uniel-uch-m111rx.json templates/config-uniel-uch-m111rx.json.jinja
$ diff -Naur ../wb-mqtt-serial-master/templates/config-uniel-uch-m111rx.json templates/config-uniel-uch-m111rx.json
--- ../wb-mqtt-serial-master/templates/config-uniel-uch-m111rx.json	2023-11-15 03:06:02.000000000 +0400
+++ templates/config-uniel-uch-m111rx.json	2023-11-15 04:53:42.000000000 +0400
@@ -102,7 +102,7 @@
                 "type": "range",
                 "max": "0xff"
             },
-            // Timers. set switch off delay
+
             {
                 "name": "Timer 0",
                 "reg_type": "param",
@@ -159,7 +159,7 @@
                 "type": "range",
                 "max": "0xff"
             },
-            // Input types. 0 = SW (normally open), 0xff = MS (normally closed)
+
             {
                 "name": "Type 0",
                 "reg_type": "relay",
@@ -200,6 +200,6 @@
                 "reg_type": "relay",
                 "address": "0x29"
             }
-        ]
+            ]
     }
 }
</code>
</details>

### CODE REVIEW

File: templates/config-ecodim-dali-gw2.json.jinja [hunk 1]
```diff
@@ -221,138 +221,58 @@
...
-                "address": "{{2002  + G * 5}}",
+                "address": "{{2002 + G * 5}}",
                 "reg_type": "holding",
                 "type": "switch",
                 "format": "u16",
                 "scale": 0.5,
                 "readonly": true,
-                "group": "group_commands",
-                "enabled": false
+                {% if G != 0 -%}
```
Comment (KraPete) [inline templates/config-ecodim-dali-gw2.json.jinja:233]: Можно в цикле сделать переменную и не делать `if` в каждом параметре, а присваивать её значение каждому `enabled`

File: templates/config-ecodim-dali-gw2.json.jinja [hunk 2]
```diff
@@ -372,143 +294,62 @@
...
             {
                 "name": "Device {{A}} Available",
                 "address": "{{3002 + A * 5}}",
                 "reg_type": "holding",
                 "type": "value",
                 "format": "u16",
                 "readonly": true,
-                "group": "individual_commands",
-                "enabled": false
+                {% if A != 0 -%}
```
Comment (KraPete) [inline templates/config-ecodim-dali-gw2.json.jinja:349]: Тут аналогично

**Review by KraPete — CHANGES_REQUESTED** [review]

File: templates/config-ecodim-dali-gw2.json.jinja (hunk 1 above)
Comment (sikmir) [reply templates/config-ecodim-dali-gw2.json.jinja:233]: поправил

File: templates/config-ecodim-dali-gw2.json.jinja (hunk 2 above)
Comment (sikmir) [reply templates/config-ecodim-dali-gw2.json.jinja:349]: поправил

**Review by KraPete — APPROVED** [review]

**Pull request merged**